### DISCUSSION TITLE
Klipperscreen resolution change?

### DISCUSSION THREAD

**Discussion by cgi2099** [KlipperScreen/KlipperScreen#940] [Q&A]
I have Ubuntu installed on a old surface 3 pad. Everything works as should including the touchscreen. Do to the linux-surface kernel.

I have used kiauh to install, klipper, moonraker, mainsail, fluidd and klipperscreen. All seems to be working great even the touch part of klipperscreen but the resolution is to big. I can only see one button and the back button. I need to set the resolution to 1920x1280, is this possible to change?

**Comment by alfrix** [comment]
resolution is handled by the OS, klipperscreen is just a fullscreen application, you need to figure out how to configure the resolution of the xserver for your particular case, sorry but i'm not familiar with your hw, if you figure this out, i encourage you to leave an answer in case others encounter need the help
